docs/source/quickstart.mdx

```diff
@@ -17,38 +17,52 @@ limitations under the License.
+Optimum Habana library is optimized for running various AI workloads on Intel® Gaudi® accelerators and it contains fully documented
+inference, training and fine-tuning examples. Please refer to the [Optimum Habana GitHub](https://github.com/huggingface/optimum-habana)
+page for more information.
 
-```diff
-- from transformers import Trainer, TrainingArguments
-+ from optimum.habana import GaudiTrainer, GaudiTrainingArguments
+## Accessing Intel® Gaudi® AI Accelerator
+To access an Intel® Gaudi® AI accelerator node in the Intel® Tiber® Developer cloud, you will go to
+[Intel Developer Cloud Console](https://console.cloud.intel.com/hardware) and access the hardware instances to select the Intel® Gaudi® AI accelerator
+platform for deep learning and follow the steps to start and connect to the node.
 
-# Define the training arguments
-- training_args = TrainingArguments(
-+ training_args = GaudiTrainingArguments(
-+   use_habana=True,
-+   use_lazy_mode=True,
-+   gaudi_config_name=gaudi_config_name,
-  ...
-)
+## Docker Setup
 
-# Initialize our Trainer
-- trainer = Trainer(
-+ trainer = GaudiTrainer(
-    model=model,
-    args=training_args,
-    train_dataset=train_dataset
-    ... # other arguments
-)
+Now that you have access to the node, you will use the latest Intel® Gaudi® AI Accelerator docker image by executing the docker run command which will
+automatically download and run the docker. At the time of writing this guide, latest Gaudi docker version was 1.18.0:
+
+```bash
+release=1.18.0
+os=ubuntu22.04
+torch=2.4.0
+docker_image=vault.habana.ai/gaudi-docker/$release/$os/habanalabs/pytorch-installer-$torch:latest
+```
+<Tip>
+
+Visit <a href="https://docs.habana.ai/en/latest/Release_Notes/GAUDI_Release_Notes.html">Intel® Gaudi® AI Accelerator Release Notes</a> page to get the latest
+Intel® Gaudi® AI accelerator software release version. Alternatively, check <a href="https://vault.habana.ai/ui/native/gaudi-docker">https://vault.habana.ai/ui/native/gaudi-docker</a>
+for the list of all released Intel® Gaudi® AI accelerator docker images.
+
+</Tip>
+
+Execute docker run command:
+```bash
+docker run -itd \
+    --name Gaudi_Docker \
+    --runtime=habana \
+    -e HABANA_VISIBLE_DEVICES=all \
+    -e OMPI_MCA_btl_vader_single_copy_mechanism=none \
+    --cap-add=sys_nice \
+    --net=host \
+    --ipc=host \
+    ${docker_image}
 ```
 
-where `gaudi_config_name` is the name of a model from the [Hub](https://huggingface.co/Habana) (Gaudi configurations are stored in model repositories) or a path to a local Gaudi configuration file (you can see [here](./package_reference/gaudi_config) how to write your own).
+where `gaudi_config_name` is the name of a model from the [Hub](https://huggingface.co/Habana) (Gaudi configurations are stored in model repositories)
```

Comment:
why do we talk about gaudi_config_name here? it seems we are talking about docker before.

Reply:
This out of place.  Good catch.  Probably leftover from previous content.

Reply:
Good catch.  This is out of place, probably leftover from previous content

Reply:
@libinta addressed suggested changes